### PR TITLE
Docker/Vagrant support

### PR DESCRIPTION
**Pull request by sandrokeil**
This is a pull request for #34 and ready to merge. Currently only MySQL is supported, but I think it's a good point to start.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,7 +34,8 @@
         "doctrine/dbal": "^2.5",
         "doctrine/migrations": "^1.0",
         "nicolopignatelli/valueobjects": "~3.0",
-        "sandrokeil/interop-config": "^0.3.1"
+        "sandrokeil/interop-config": "^0.3.1",
+        "prooph/event-store-doctrine-adapter": "^3.0"
```
Comment (codeliner) [inline composer.json]: No adapter should be installed by default to force the user to choose one. Please remove

**Comment by codeliner** [comment]
@sandrokeil Thanks for this great addition! Just a small change required then I'll merge it.

**Comment by sandrokeil** [comment]
@codeliner Gotcha! Sorry, I've fixed it.

**Comment by codeliner** [comment]
:+1:

**Pull request merged**